lib/modules/orangered.js

```diff
@@ -196,6 +196,9 @@ const setupFaviconBadge = _.once(() => {
 
 	const favicon = new Favico();
 
+	// remove other icons, since Chrome (bizarrely) switches to one of them if RES changes the favicon
+	for (const e of document.head.querySelectorAll('link[rel="icon"]')) e.remove();
```

Comment:
Maybe move this above `new Favico()` just in case its initializer tries to sniff the existing icons?